addons-cluster/redis/templates/_helpers.tpl

```diff
@@ -243,13 +262,16 @@ metadata:
   name: {{ include "kblib.clusterName" . }}
   namespace: {{ .Release.Namespace }}
   labels: {{ include "kblib.clusterLabels" . | nindent 4 }}
-  {{- if and .Values.hostNetworkEnabled (eq .Values.mode "cluster") }}
   annotations:
+    apps.kubeblocks.io/mode: {{ .Values.mode }}
```

Comment:
Is the annotation still needed?

Reply:
`apps.kubeblocks.io/mode` for custom web console fliter